style_guide.md

```diff
@@ -0,0 +1,307 @@
+- `questionTitle` returns the appropriate JSON for title.
+
+- Same rules apply to other blocks with variants, e.g. for guidance with a `questionGuidance` method we would use:
+    ```
+    local questionGuidance(isProxy) = (
+      if isProxy then 
+        'Why we ask this question if they are retired or long-term sick or disabled' 
+      else 'Why we ask this question if you are retired or long-term sick or disabled';
+    );
+    ```
+#### Multiple value variants example
+
+- There are occasions where more than one variable is used to create variants. Example of this would be a `Question` block using combination of `isProxy` and `isEmployed` variables:
+
+    ```
+    {
+      type: 'Question',
+      id: 'business-name',
+      question_variants: [
+        {
+          question: question(isProxy=false, isEmployed=true),
+          when: [rules.isNotProxy, rules.isEmployed],
+        },
+        {
+          question: question(isProxy=true, isEmployed=true),
+          when: [rules.isProxy, rules.isEmployed],
+        },
+        {
+          question: question(isProxy=false, isEmployed=false),
+          when: [rules.isNotProxy, rules.isNotEmployed],
+        },
+        {
+          question: question(isProxy=true, isEmployed=false),
+          when: [rules.isProxy, rules.isNotEmployed],
+        },
+      ],
+    };
+  
+    local question(isProxy, isEmployed) = {
+      id: 'business-name-question',
+      title: questionTitle(isProxy, isEmployed),
+      description: description,
+      type: 'MutuallyExclusive',
+      mandatory: false,
+      answers: [
+        {
+          id: 'business-name-answer',
+          label: 'Organisation or business name',
+          max_length: 160,
+          mandatory: false,
+          type: 'TextField',
+        },
+        {
+          id: 'no-business-name-answer',
+          mandatory: false,
+          type: 'Checkbox',
+          options: [
+            {
+              label: option,
+              value: option,
+            },
+          ],
+        },
+      ],
+    };
+  
+    local questionTitle(isProxy, isEmployed) = (
+      if isEmployed then (
+        if isProxy then 
+          {
+            text: 'What is the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
+            placeholders: [
+              placeholders.personNamePossessive,
+            ]
+          }
+        else 'What is the main activity of your organisation, business or freelance work?'
+      ) else (
+        if isProxy then 
+          {
+            text: 'What was the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
+            placeholders: [
+              placeholders.personNamePossessive,
+            ]
+          }
+        else 'What was the main activity of your organisation, business or freelance work?';
+      )
+    );
+    ```
```

Comment:
Same as the previous suggestion on ordering:
```suggestion
- There are occasions where more than one variable is used to create variants. For example:

    ```
    local questionTitle(isProxy, isEmployed) = (
      if isEmployed then (
        if isProxy then 
          {
            text: 'What is the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
            placeholders: [
              placeholders.personNamePossessive,
            ]
          }
        else 'What is the main activity of your organisation, business or freelance work?'
      ) else (
        if isProxy then 
          {
            text: 'What was the main activity of <em>{person_name_possessive}</em> organisation, business or freelance work?',
            placeholders: [
              placeholders.personNamePossessive,
            ]
          }
        else 'What was the main activity of your organisation, business or freelance work?';
      )
    );

    local question(isProxy, isEmployed) = {
      id: 'business-name-question',
      title: questionTitle(isProxy, isEmployed),
      description: description,
      type: 'MutuallyExclusive',
      mandatory: false,
      answers: [
        {
          id: 'business-name-answer',
          label: 'Organisation or business name',
          max_length: 160,
          mandatory: false,
          type: 'TextField',
        },
        {
          id: 'no-business-name-answer',
          mandatory: false,
          type: 'Checkbox',
          options: [
            {
              label: option,
              value: option,
            },
          ],
        },
      ],
    };

    {
      type: 'Question',
      id: 'business-name',
      question_variants: [
        {
          question: question(isProxy=false, isEmployed=true),
          when: [rules.isNotProxy, rules.isEmployed],
        },
        {
          question: question(isProxy=true, isEmployed=true),
          when: [rules.isProxy, rules.isEmployed],
        },
        {
          question: question(isProxy=false, isEmployed=false),
          when: [rules.isNotProxy, rules.isNotEmployed],
        },
        {
          question: question(isProxy=true, isEmployed=false),
          when: [rules.isProxy, rules.isNotEmployed],
        },
      ],
    };
    ```
```